src/vs/code/electron-sandbox/issue/issueReporterService.ts

```diff
@@ -665,7 +666,7 @@ export class IssueReporter extends Disposable {
 
 		sourceSelect.innerText = '';
 		sourceSelect.append(this.makeOption('', localize('selectSource', "Select source"), true));
-		sourceSelect.append(this.makeOption('vscode', localize('vscode', "Visual Studio Code"), false));
+		sourceSelect.append(this.makeOption('azuredatastudio', issueReporterMainAzuredatastudio, false)); // {{SQL CARBON EDIT}} Update name
```

Comment:
how is the option value 'azuredatastudio' used later when creating the issue?

Reply:
it doesn't look like it's gets used later (same with the previous option value of `vscode`), but if the option `extension` is selected, then it tries to look for similar issues and list those.

Reply:
Took a quick look myself and seems like that one isn't used - just the other two are (extension and marketplace)

https://github.com/microsoft/azuredatastudio/blob/620f5f12e7aefbf3e50a622cb7802de5a38a00bb/src/vs/code/electron-sandbox/issue/issueReporterService.ts#L293